flow/project.py

```diff
@@ -86,48 +86,58 @@
 {template_vars}
 
 Filter functions can be used to format template variables in a specific way.
-For example: {{ project.get_id() | captialize }}.
+For example: {{ project.get_id() | capitalize }}.
 
 The available filters are:
 {filters}"""
 
 
 class _condition(object):
-
-    def __init__(self, condition):
+    # Random tag id used to ensure that "always" and "never" conditions never
```

Comment:
This is more of a clarifying question, but I don't see how the the tag id is 'random'. I think it either needs a better name (arbitrary?) or the comment here needs to explain.

Reply:
You're correct, arbitrary is a more appropriate choice of words. I'll change that name.

Reply:
For clarification, it only exists to ensure that these conditions (always and never) are not found to be equal.